src/bazel-tsconfig-test.json

```diff
@@ -4,7 +4,7 @@
 {
   "extends": "./bazel-tsconfig-build.json",
   "compilerOptions": {
-    "importHelpers": false,
+    "importHelpers": true,
```

Comment:
This needs to be set to true because the inline emitted helpers by TypeScript are currently broken in TS 3.9.2. A fix has landed, but will be available in the next patch.. 

We can leave this enabled, or turn it back to `false`. Both have pros and cons.

Reply:
FWIW: The CLI has `importHelpers: true` for tests too.